src/main/java/nl/novi/techiteasy/controllers/TelevisionController.java

```diff
@@ -1,71 +1,78 @@
 package nl.novi.techiteasy.controllers;
+import jakarta.validation.Valid;
+import nl.novi.techiteasy.dtos.TelevisionInputDto;
+import nl.novi.techiteasy.dtos.TelevisionOutPutDto;
 import nl.novi.techiteasy.exceptions.RecordNotFoundException;
-import nl.novi.techiteasy.models.CreateTvRequest;
-import nl.novi.techiteasy.models.Television;
-import org.springframework.http.HttpStatus;
+import nl.novi.techiteasy.services.TelevisionService;
 import org.springframework.http.ResponseEntity;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.FieldError;
 import org.springframework.web.bind.annotation.*;
-import java.util.ArrayList;
-import java.util.Iterator;
-import java.util.Optional;
-import java.util.concurrent.atomic.AtomicInteger;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+import java.net.URI;
+import java.util.List;
+
+
 
 @RestController
 @RequestMapping("/televisions")
 public class TelevisionController {
-    private final ArrayList<Television> televisionDataBase = new ArrayList<>();
-    private final AtomicInteger idCounter = new AtomicInteger(1);
-    @PostMapping
-    public ResponseEntity<Television> create(@RequestBody CreateTvRequest model) {
-        int id = idCounter.getAndIncrement();
-        Television result = new Television(id, model.name, model.price);
-        this.televisionDataBase.add(result);
-        return ResponseEntity.created(null).body(result);
-    }
 
-    @GetMapping("/get")
-    public ResponseEntity<Television> get() {
-        if(this.televisionDataBase.isEmpty()) {
-            throw new RecordNotFoundException("Television not found");
-        }
-        else {
-            return ResponseEntity.ok(this.televisionDataBase.get(0));
-        }
-    }
+    private final TelevisionService service;
+
+    public TelevisionController(TelevisionService televisionService){this.service = televisionService;}
 
     @GetMapping
-    public ResponseEntity<ArrayList<Television>> getAll() {
+    public ResponseEntity<List<TelevisionOutPutDto>> getAllTelevision() {
+        return ResponseEntity.ok(service.GetAllTelevision());
```

Comment:
ik zou GetAllTelevision zonder hoofdletter gebruiken dus getAllTelevision

Reply:
inderdaad thx dat is wel beter